package.json

```diff
@@ -44,13 +44,13 @@
         "@babel/plugin-proposal-class-properties": "^7.5.5",
         "@babel/plugin-proposal-export-default-from": "^7.5.2",
         "@babel/plugin-transform-arrow-functions": "^7.2.0",
-        "@storybook/addon-a11y": "^6.0.20",
-        "@storybook/addon-actions": "^6.0.20",
-        "@storybook/addon-docs": "^6.0.20",
-        "@storybook/addon-essentials": "^6.0.20",
-        "@storybook/addon-knobs": "^6.0.20",
-        "@storybook/addon-links": "^6.0.20",
-        "@storybook/react": "^6.0.20",
+        "@storybook/addon-a11y": "^6.0.27",
```

Comment:
shouldn't all of this already get pulled in for anyone who is using it because caret versioning?

Reply:
Possibly! I just ran the command that Storybook was throwing me a warning for. It probably means that our hosted Storybook (since it re-installs dependencies each time) has the latest and greatest already...